### PR TITLE
Fix version handling

### PR DESCRIPTION
**Pull request by wallento**

### CODE REVIEW

**Comment by wallento** [comment]
The test fails because it seems that the packaging module does not contain the "is_devrelease" property on version in older releases. Not really sure how to handle it, @imphil. Straightforward solution is to strip away (\.dev.*) in a regexp.

File: tools/build.py [hunk 1]
```diff
@@ -174,22 +174,26 @@ def check_autotools():
...
+              .format(min_version, found_version_str))
     else:
-        dbg("Found FuseSoC version " + found_version)
+        dbg("Found FuseSoC version " + found_version_str)
 
 """Check for vivado
 """
 def check_vivado(min_version):
     ver_str = check_program('vivado', '-version')
     # XXX: most likely doesn't cover service releases, extend if we see one of
```
Comment (imphil) [inline tools/build.py]: Please remove that now-outdated comment.

Comment (wallento) [reply tools/build.py]: Done in force push

**Comment by wallento** [comment]
Problems are resolved after force-push, can be merged

**Pull request merged**